boards/FSM/Inc/App/configs/App_FlowRateThresholds.h

```diff
@@ -2,7 +2,7 @@
 
 // The datasheet (https://www.adafruit.com/product/828) for the flow meter
 // indicates a working flow rate between 1L/min to 30L/min
-#define PRIMARY_FLOW_METER_MIN_FLOW_RATE 1.0f
-#define PRIMARY_FLOW_METER_MAX_FLOW_RATE 30.0f
-#define SECONDARY_FLOW_METER_MIN_FLOW_RATE 1.0f
-#define SECONDARY_FLOW_METER_MAX_FLOW_RATE 30.0f
+#define PRIMARY_FLOW_METER_MIN_FLOW_RATE_L_PER_MIN 1.0f
+#define PRIMARY_FLOW_METER_MAX_FLOW_RATE_L_PER_MIN 30.0f
+#define SECONDARY_FLOW_METER_MIN_FLOW_RATE_L_PER_MIN 1.0f
+#define SECONDARY_FLOW_METER_MAX_FLOW_RATE_L_PER_MIN 30.0f
```

Comment:
More consistent and concise this way
```suggestion
#define MIN_PRIMARY_FLOW_RATE_L_PER_MIN 1.0f
#define MAX_PRIMARY_FLOW_RATE_L_PER_MIN 30.0f
#define MIN_SECONDARY_FLOW_RATE_L_PER_MIN 1.0f
#define MAX_SECONDARY_FLOW_RATE_L_PER_MIN 30.0f
```